### ISSUE TITLE
"Vershbow" vice "Vershow"

### ISSUE THREAD

**Issue by therealjameswilson** [HistoryAtState/frus#304]
I believe should be "Vershbow" (for Alexander "Sandy" Vershbow) -- surely my own mistake here

https://history.state.gov/historicaldocuments/frus1981-88v11/d324

**Comment by joewiz** [comment]
@therealjameswilson There's one other instance of "Vershow" in FRUS: https://history.state.gov/search?q=vershow. Do you believe the other is a typo too?

**Comment by therealjameswilson** [comment]
Yeah, it’s definitely Sandy Vershbow, who was later ambassador to Russia and South Korea.
